tidb-cloud/use-chat2query-api.md

```diff
@@ -84,6 +84,10 @@ TiDB Cloud generates code examples to help you call an endpoint. To get the exam
     - Replace the `<your instruction>` placeholder with the instruction you want AI to generate and execute SQL statements.
     - Replace the `<your table name, optional>` placeholder with the table name you want to query. If you do not specify a table name, AI will query all tables in the database.
 
+> **Note:**
```

Comment:
{
"type":"chat2data_endpoint",
"data": {
        "columns":[
        ],
        "rows":[
        ]
        "result":{
            "code": 429,
            "message": "The AI request exceeded the limit of 100 times per day. For more quota, please contact us: https://support.pingcap.com/hc/en-us/requests/new?ticket_form_id=7800003722519",
            "start_ms": "",
            "end_ms": "",
            "latency": "",
            "row_count": 0,
            "row_affect": 0,
            "limit": 0,
            "query": ""
        }
    }
}